volatility/framework/plugins/linux/check_modules.py

```diff
@@ -0,0 +1,87 @@
+# Volatility
```

Comment:
We don't typically include these two lines in core, we just launch straight into the copyright, but it's not a show stopper.  5:)